### PR TITLE
Tuya RB-SRAIN01: change 'water-leak' to 'rain'; remove 'battery_low' and 'tamper'

### PR DESCRIPTION
**Pull request by marcelhoogantink**
Changed 'water_leak' expose to 'rain' to make more sense for this device, also replaced fz.ias_water_leak_alaram_1 into local option, to accept 'rain' in stead of 'water_leak' and to remove 'tamper' and 'battery_low' since these last two don't exist in RB_SRAIN01

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -1896,10 +1896,23 @@ const definitions: DefinitionWithExtend[] = [
         model: 'RB-SRAIN01',
         vendor: 'Tuya',
         description: 'Solar rain sensor',
-        fromZigbee: [tuya.fz.datapoints, fz.battery, fz.ias_water_leak_alarm_1],
+        fromZigbee: [
+            tuya.fz.datapoints,
+            fz.battery,
+            {
```
[the remark this answers is not in the document]
Comment (marcelhoogantink) [reply src/devices/tuya.ts:1902]: So, I tried, but this gives many unneeded items in payload:
```
{
    "ac_status": false,               // unneeded
    "battery_defect": false,      // unneeded
    "battery_low": false,          // unneeded
    "linkquality": 224,
    "restore_reports": false,        // unneeded
    "supervision_reports": false, // unneeded
    "tamper": false,                    // unneeded
    "test": false,                         // unneeded
    "trouble": false,                   // unneeded
    "battery": 90,
    "illuminance": 120,
    "illuminance_average_20min": 3369,
    "illuminance_maximum_today": 4103,
    "cleaning_reminder": true,
    "rain_intensity": 44,
    "rain": false
}
```
How to remove those unneeded items?

[the remark this answers is not in the document]
Comment (Koenkk) [reply src/devices/tuya.ts]: I propose to leave them there, they can be useful for debugging and this keeps it consistent with other devices.

[the remark this answers is not in the document]
Comment (marcelhoogantink) [reply src/devices/tuya.ts]: Ready

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**